develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,181 @@
+
+          ```json
+          { 
+            "parents": 0, 
+            "interior": "Here"
+          }
+          ```
+
+      4. Set the **fun** parameter to `true` to identify the asset as a fungible asset
+      5. Specify the **fungible** parameter to indicate the total fungible assets to withdraw. For example, this tutorial uses `12000000000000`
+      6. Select `Unlimited` to skip setting a weight limit for this instruction on the **weightLimit** parameter
+
+        ![](/images/tutorials/polkadot-sdk/build-a-parachain/transfer-assets-with-xcm/transfer-assets-with-xcm-6.webp)
+
+## DepositAsset Instruction
+
+To deposit assets after fees from the holding register into a specific account:
+
+1. Prepare the [DepositAsset](https://github.com/polkadot-fellows/xcm-format#depositasset){target=\_blank} as the third instruction for this message
+      1. Click on **+ Add item** to add the instruction to the message
+      2. Select **DepositAsset** from the dropdown menu
+      3. For the **asset** parameter, set `Wild` to allow an unspecified number of assets to be deposited. Then, set `All` for the *wild* parameter to allow all of the remaining assets after fees are paid to be deposited
+      4. For the **maxAssets** parameter, set `1` as the maximum number of unique assets to remove from the holding register for the deposit. In this tutorial, there's only one asset instance available to be removed
+      5. For the **beneficiary** parameter, set the destination account for the deposit. In this case, the account is located in parachain A, so the value looks like this:
+
+         ```json
+         {
+           "parents": 0,
+           "interior": {
+             "X1": {
+               "junction": {
+                 "AccountId32": {
+                   "network": "Any",
+                   "id": "5DEy8ViuBxh8z8hxeNyBhj4y5Mz2sNybv2aPqDHBh7EsyRAz"
+               }
+             }
+           }
+         }
+         ```
+
+        ![](/images/tutorials/polkadot-sdk/build-a-parachain/transfer-assets-with-xcm/transfer-assets-with-xcm-7.webp)
+
+        !!! note
+            If you want to deposit the assets into the sovereign account for parachain A, you could specify the beneficiary using the following settings:
+                  ```json
+                  {
+                    "parents": 0,
+                    "interior": {
+                      "X1": {
+                        "junction": "Parachain",
+                        "parachainIndex": 1000
+                      }
+                    }
+                  }
+                  ```
+After you configure all of the XCM instructions to be executed, you're ready to submit the transaction.
+
+## Submit and Check the Transaction
+
+After submitting the transaction, you can check the status of the transaction in the **Explorer** tab on the parachain B interface. If the transaction is successful, you should see the `polkadotXcm.Sent` event in the **Events** tab.
```

Comment:
The `sent` extrinsic was successfully executed, but I think the actual transfer could have failed in the remote chain